freckle-app.cabal

```diff
@@ -140,6 +141,7 @@ library
     , errors
     , exceptions
     , extra
+    , faktory
```

Comment:
Ah, I didn't realize *none* of our existing Faktory wrapper stuff was here yet. Now I remember.

https://app.asana.com/0/1204746045920874/1200495324837436/f

Stuff like `HasFaktorySettings`, `enqueueSink` maybe, could all be brought in here.

Reply:
That was why I was hesitant to bring this over here yeah- it was pretty annoying to do the dev with it being split up, but I wasn't exactly sure which pieces we would want to move over. 

I guess anything that both `jobs` and `fancy-api` uses would make sense as a starting point.

Reply:
> I guess anything that both jobs and fancy-api uses would make sense as a starting point

Anything that's business-agnostic is fair game for freckle-app. That sharing is a good signal for prioritizing though; in case we ever want to pull out `jobs` code into something stream-aligned, it would be useful to have those bits extracted for sure.

Reply:
I do think all of this logic could live in `faktory` too, but there are some questions there. That library is simplistic in terms of being concretely `IO`. If we're going to start talking about `MonadReader env m, HasSomething env` within that library, we might want to consider migrate it to a more abstract monad (which would open up additional niceties like relying on `MonadLogger`, `MonadTrace`, etc).